requirements.txt

```diff
@@ -0,0 +1,6 @@
+requests~=2.26.0
+pycryptodome==3.11.0
+dicttoxml==1.7.4
+requests_cache==0.9.1
+pycryptodome==3.11.0
+deepmerge==1.0.1
```

Comment:
Pycharm complained that some dependencies from "Crypto" are not included here.
Try: `pip freeze > requirements.txt` to ensure all the dependencies are inclueded